pygmt_helper/pygmt_helper/plotting.py

```diff
@@ -236,6 +243,10 @@ def plot_grid(
     log_cmap: bool, optional
         Create a log10 based colourmap
         Expects the cmap_limits to be log10(z)
+    transparency: float, optional
+        Controls the level of transparency (0-100)
+    plot_contours: bool, optional
+    continuous_cpt: bool, optional
```

Comment:
The `plot_contours` and `continuous_cpt` arguments need descriptions. I don't know what `continuous_cpt` means.